crates/db_connection_pool/src/postgrespool.rs

```diff
@@ -182,6 +182,7 @@ impl PostgresConnectionPool {
 
         let pool = bb8::Pool::builder()
             .error_sink(Box::new(error_sink))
+            .test_on_check_out(false)
```

Comment:
Not sure why this is added.

Reply:
Sry for letting that slip in. It's needed to connect to the posgres emulator of https://github.com/edgedb/edgedb 
Otherwise, the bb8 connection check times out. I think edgedb's emulation doesn't like the test query.

So in effect, can be removed. It could also stay since further down, we do another check anyway.

Reply:
thanks for the clarification. I will remove it for now. Also TIL edgedb :)